helm/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: immudb
+description: The immutable database
+type: application
+version: 0.1.0
+appVersion: "1.1.0"
```

Comment:
Please update the `/build/RELEASING.md` file with info to also bump this version here if a new immudb release is made.

Reply:
fixed in commit 3cad64d8